applications/evaluation/Evaluation.csproj

```
@@ -0,0 +1,23 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+    <PropertyGroup>
+        <TargetFramework>net8.0</TargetFramework>
+        <AssemblyName>Microsoft.KernelMemory.Evaluation</AssemblyName>
+        <RootNamespace>Microsoft.KernelMemory</RootNamespace>
+        <NoWarn>$(NoWarn);KMEXP00;CA1711;CS1591;CS1574;NU5104;SKEXP0001;</NoWarn>
+    </PropertyGroup>
+
+    <PropertyGroup>
+        <IsPackable>true</IsPackable>
```

```suggestion
        <IsPackable>false</IsPackable>
```
This would generate a new nuget, but there's no code yet.